packages/contracts-bedrock/scripts/deploy/ChainAssertions.sol

```diff
@@ -567,23 +569,79 @@ library ChainAssertions {
         }
     }
 
-    /// @notice Asserts that the SuperchainConfig is setup correctly
-    function checkOPContractsManager(Types.ContractSet memory _contracts, bool _isProxy) internal view {
-        OPContractsManager opcm = OPContractsManager(_contracts.OPContractsManager);
-        console.log(
-            "Running chain assertions on the OPContractsManager %s at %s",
-            _isProxy ? "proxy" : "implementation",
-            address(opcm)
+    /// @notice Asserts that the OPContractsManager is setup correctly
+    function checkOPContractsManager(
+        Types.ContractSet memory _contracts,
```

Comment:
nit: I know it's not in your diff, but unlike the other assertions methods, this is always meant to be called with `_contracts = _impls()`, so maybe we should rename the arg to clear about that? I think it'd make the checks in the function body a bit more 'symmetrical':
```diff
- require(impls.l1ERC721BridgeImpl == _contracts.L1ERC721Bridge, "CHECK-OPCM-50");
+ require(impls.l1ERC721BridgeImpl == _impls.L1ERC721Bridge, "CHECK-OPCM-50");
```

Reply:
resolving this, not a big deal.